frontend/src/components/ExtendedPlantDisplay.tsx

```diff
@@ -0,0 +1,29 @@
+import { PlantsSummaryDto } from '@/bindings/definitions';
+import { ExtendedPlantsSummary } from '@/utils/ExtendedPlantsSummary';
+
+const ExtendedPlantsSummaryDisplayName = ({ plant }: { plant: PlantsSummaryDto }) => {
+  const transformedPlant = new ExtendedPlantsSummary(plant);
+
+  return (
+    <span>
+      {transformedPlant.displayName.common_name ? (
+        <>
+          {transformedPlant.displayName.common_name} (
```

Comment:
Great job to have this code at a single spot. Please also add additional name here, see doc/database/guidelines.md (Could be follow up but I would prefer if it is already done here, as it is difficult to test without being able to read the additional name).